dev/RadialGradientBrush/RadialGradientBrush.idl

```diff
@@ -27,13 +27,18 @@ unsealed runtimeclass RadialGradientBrush : Windows.UI.Xaml.Media.XamlCompositio
     [MUX_DEFAULT_VALUE("winrt::Windows::UI::Composition::CompositionColorSpace::Auto")]
     Windows.UI.Composition.CompositionColorSpace InterpolationSpace { get; set; };
 
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    [MUX_DEFAULT_VALUE("winrt::GradientSpreadMethod::Pad")]
+    Windows.UI.Xaml.Media.GradientSpreadMethod SpreadMethod { get; set; };
```

Comment:
Has this control gone through API review?

Reply:
It's in progress